### PR TITLE
COVID Stimulus Watch job

### PR DESCRIPTION
**Pull request by timsehn**
Why does it need to be more complicated than this?

### CODE REVIEW

File: airflow_dags/covid_stimulus_watch/import-csv.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    'Median Worker Pay': 'median_worker_pay',
+    'Parent TARP Loans Received During Financial Crisis': 'parent_tarp_loans'
+}
+
+pks = ['recipient_company', 'parent_company']
+
+url = 'https://data.covidstimuluswatch.org/prog.php?&detail=export_csv'
+outcsvfile = 'dolt-import.csv'
+
+tmpdir = '.'
```
Comment (oscarbatori) [inline airflow_dags/covid_stimulus_watch/import-csv.py:50]: You can use `tmpdir` for this, it's a Python library that gives you a temporary directory for use and cleans up afterwards.

File: airflow_dags/covid_stimulus_watch/import-csv.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
+#!/usr/local/bin/python3
+
+import csv
+import datetime
+import urllib.request
+import io
+
```
Comment (oscarbatori) [inline airflow_dags/covid_stimulus_watch/import-csv.py:7]:
```
import logging

logger = logging.getLogger(__name__)
```

File: airflow_dags/covid_stimulus_watch/import-csv.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+org = 'Liquidata'
+repo_name = 'covid-stimulus-watch'
+
+target = f'{org}/{repo_name}'
+
+print(f'Cloning {target}')
+repo = clone_repo(target, '.')
+
+print(f'Reading {url}')
```
Comment (oscarbatori) [inline airflow_dags/covid_stimulus_watch/import-csv.py:60]: See above for defining `logger`:
```
logger.info('...')
```

File: airflow_dags/covid_stimulus_watch/import-csv.py [hunk 4]
```diff
@@ -0,0 +1,95 @@
+#!/usr/local/bin/python3
+
+import csv
+import datetime
+import urllib.request
```
Comment (oscarbatori) [inline airflow_dags/covid_stimulus_watch/import-csv.py:5]: Use requests instead, then:
```
import requests
```

File: airflow_dags/covid_stimulus_watch/import-csv.py (hunk 1 above)
Comment (timsehn) [reply airflow_dags/covid_stimulus_watch/import-csv.py:50]: Airflow handles that for me :-)

**Pull request merged**